Source/Parsers/SVGKParseResult.h

```diff
@@ -17,8 +17,8 @@
 /** 0.0 = no parsing done yet, 0.x = partially parsed, 1.0 = parse complete (no fatal errors) */
 @property(nonatomic) double parseProgressFractionApproximate;
 
-@property(nonatomic,strong) SVGSVGElement* rootOfSVGTree; /*< both are needed, see spec */
-@property(nonatomic,strong) SVGDocument* parsedDocument; /*< both are needed, see spec */
+@property(nonatomic,strong) SVGSVGElement* rootOfSVGTree; /**< both are needed, see spec */
```

Comment:
Just Xcode warning. Because Xcode now parse all comment as [Doxygen](https://www.stack.nl/~dimitri/doxygen/manual/docblocks.html) format. (I don't know whether Xcode 6 or old version behavior, but when I become iOS developer I use Xcode 7).

The `<` is a doxygen syntax, means the comment should appear above the thing (such as property). So Xcode will warning you...Simple way is to use `/** < This comment is above property */`.


![image](https://user-images.githubusercontent.com/6919743/46905471-dc81c880-cf26-11e8-8e6f-1f41cd27914c.png)

Reply:
Ah, OK, so it looks like the standard has changed :). Fine.

Reply:
Actually...You can just merge this PR: #578  He do the same thing as me...

I'll rebase local branch after that PR merged. To simplify diff changes.